### ISSUE TITLE
1.27 introduces test failure t/nonexistent-locale.t line 24

### ISSUE THREAD

**Issue by kentfredric** [houseabsolute/DateTime-Locale#27]
```
===(     703;5  665/?  1/?  1/? )=======================================
# Failed test 'got an exception trying to read an unreadable file'
# at t/nonexistent-locale.t line 24.
# +--------------------------------+----+--------------------------------+
# | GOT                            | OP | CHECK                          |
# +--------------------------------+----+--------------------------------+
# | File '/var/tmp/portage/dev-per | =~ | (?^:No read permission)        |
# | l/DateTime-Locale-1.270.0/temp |    |                                |
# | /iPMqstUq4y/auto/share/dist/Da |    |                                |
# | teTime-Locale/unreadable.pl',  |    |                                |
# | no read permissions at /var/tm |    |                                |
# | p/portage/dev-perl/DateTime-Lo |    |                                |
# | cale-1.270.0/work/DateTime-Loc |    |                                |
# | ale-1.27/blib/lib/DateTime/Loc |    |                                |
# | ale/Data.pm line 6094.\n       |    |                                |
# +--------------------------------+----+--------------------------------+
# Seeded srand with seed '20200829' from local date.
t/nonexistent-locale.t ........... Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
```

**Comment by kentfredric** [comment]
And a few cpan testers replications (all by SREZIC)

- http://www.cpantesters.org/cpan/report/7e6e9f12-e95e-11ea-9b31-bf311f24ea8f
- http://www.cpantesters.org/cpan/report/6814bb66-e95e-11ea-a553-9a311f24ea8f
- http://www.cpantesters.org/cpan/report/d8d3fb98-e961-11ea-9f4f-cb661f24ea8f
- http://www.cpantesters.org/cpan/report/1aad1e18-e95e-11ea-8e2e-732f1f24ea8f

**Comment by kentfredric** [comment]
https://github.com/houseabsolute/DateTime-Locale/blob/d727eb79700f8996faaff92e58f7d8e1ae010aa5/t/nonexistent-locale.t#L24

Well, yeah, that explains things.

**Comment by kentfredric** [comment]
Failure of course introduced by #24

**Comment by kentfredric** [comment]
Ah, my mistake, not added in the PR, but added inline in the merge as the test. 

https://github.com/houseabsolute/DateTime-Locale/commit/3cdc7ce49e07994324cab85633f1aec9f177d5e6

**Comment by autarch** [comment]
Thanks for the heads up. I released 1.28 to fix this. I just moved the test to xt because it's not terribly important, and fixing it reliably for everyone isn't worth the effort.
